docs/components/board/_index.md

```diff
@@ -24,9 +24,14 @@ The [RDK](/internals/rdk) also provides the [`GPIOPin` interface](#gpiopin-api)
 
 ## Configuration
 
-Configure a *board* component on your robot to communicate with the other [components](/components) of the robot, signaling as overseen by a computer running `viam-server`.
+Configure a *board* component on your robot to communicate with the other [components](/components) of the robot.
+Signaling is overseen by a computer running `viam-server`.
```

Comment:
Hmmm-- sorry to be a bother here but I think splitting this out makes the sentence flow a bit more awkward as it's supposed to be kind of cause and effect!

Reply:
Up to you if you want to keep or not, LGTM either way